app/routes/campaigns.js

```diff
@@ -199,6 +200,23 @@ router.post('/:id/message', (req, res, next) => {
 });
 
 /**
+ * Load the user to track outgoing message.
+ */
+router.post('/:id/message', (req, res, next) => {
+  User.lookup('mobile', req.body.phone)
+    .then((user) => {
+      if (req.timedout) {
+        return helpers.sendTimeoutResponse(res);
+      }
+
+      user.postDashbotOutgoing(req.messageType);
+
+      return next();
+    })
+    .catch(err => helpers.sendErrorResponse(res, err));
```

Comment:
This was added to log the analytics to a User's Northstar ID instead of phone in third party. 

Nothing urgent, but heads up that we've just seen 2 instance of 404's for Users not found by their number, which is odd because if they're receiving a relative reminder message - they should exist in Northstar with a number already.

cc @sergii-tkachenko @rapala61